.cron.yml

```diff
@@ -10,3 +10,9 @@ jobs:
           target-tasks-method: l10n_screenshots
           treeherder-symbol: l10-screenshots
       when: []  # Manual trigger only
+    - name: tab-performance-test
+      job:
+          type: decision-task
+          target-tasks-method: tab_performance_test
+          treeherder-symbol: tab-performance-test
+      when: []  # Manual trigger only
```

Comment:
https://github.com/mozilla-mobile/firefox-android/blob/a536d1d1d03d34778830dc02b8882bce8c3e6357/.cron.yml#L22

Reply:
updated to have the test completed sometime between 6-7am gmt+2